libbeat/processors/add_cloud_metadata/add_cloud_metadata.go

```diff
@@ -74,78 +81,108 @@ var (
 
 		return out
 	}
+
+	qcloudSchema = func(m map[string]interface{}) common.MapStr {
+		out, _ := s.Schema{
+			"instance_id":       c.Str("instance_id"),
+			"region":            c.Str("region"),
+			"availability_zone": c.Str("zone"),
+		}.Apply(m)
+		return out
```

Comment:
It seems like this doesn't even need to use the `Schema` as you have already built up the `map[string]interface{}` with the proper key names and data types. So I think this could just become `return common.MapStr(m)`.